incubator/exporter-kubernetes/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
```

Comment:
In the future, we should set the `description` to something non-generic =) I've added an issue to track this.